File: jenjin-core/src/main/java/com/binarymonks/jj/JJConfig.java
package com.binarymonks.jj;

public class JJConfig {
    public boolean b2dDebug = false;
    public GameViewConfig gameViewConfig = new GameViewConfig();

    public static class GameViewConfig{
        public float worldBoxWidth=100;
        public float cameraPosX;
        public float cameraPosY;
    }
}

File: jenjin-core/src/main/kotlin/com/binarymonks/jj/core/JJConfig.kt
package com.binarymonks.jj.core

import com.badlogic.gdx.math.Vector2

class JJConfig {
    var gameView = GameViewConfig()
    var b2d = B2DConfig()
    var spine = SpineConfig()
}

class GameViewConfig {
    var worldBoxWidth = 50f
    var cameraPosX: Float = 0f
    var cameraPosY: Float = 0f
}

class B2DConfig {
    var debug = false
    var gravity = Vector2(0f, -9f)
}

class SpineConfig {
    var render = true
}